app/controllers/admin_comment_controller.rb

```diff
@@ -29,15 +29,20 @@ def edit
   def update
```

Comment:
I'll leave this here as a reminder for now (otherwise the history gets too messy to deal with), tempted to split the commit into what that says and adding the admin functionality afterwards